src/main/java/uk/gov/hmcts/probate/model/ccd/raw/request/CaseDataParent.java

```diff
@@ -17,6 +17,8 @@
 public class CaseDataParent {
 
     protected String schemaVersion;
+    // A second copy of schemaVersion, holding exactly the same value.
+    // Needed tdue to ccd quirks/RI to allow its use in FieldShowCondition for multiple pages for same event
```

Comment:
Was there supposed to be a schemaVersionCcdCopy field in this class?

Reply:
yes will attend!